crates/aptos/src/move_tool/mod.rs

```diff
@@ -319,6 +319,16 @@ impl CliCommand<&'static str> for ProvePackage {
             install_dir: self.move_options.output_dir.clone(),
             ..Default::default()
         };
+
+        const APTOS_NATIVE_TEMPLATE: &[u8] = include_bytes!("aptos-natives.bpl");
+
+        let mut options = move_prover::cli::Options::default();
```

Comment:
We'll need to explain how to install boogie in the docs at some point

Reply:
Sure, we have the instruction at https://github.com/move-language/move/blob/main/language/move-prover/doc/user/install.md, which may not be well visible to the Aptos users though.

Please let me know which Aptos docs I can update about this.